pkg/services/baremetal/host/host.go

```diff
@@ -45,7 +45,7 @@ import (
 )
 
 const (
-	rebootWaitTime       time.Duration = 15 * time.Second
+	rebootWaitTime       time.Duration = 30 * time.Second
```

Comment:
are you sure that we want to increase this? This is about the time the server needs to start the reboot. AFAIK this should be done in a few seconds

Reply:
ok, I will revert my change.